std_msgs/Float64 fx
std_msgs/Float64 fy
std_msgs/Float64 cu
std_msgs/Float64 cv
std_msgs/Bool isDetected
std_msgs/Float64 u
std_msgs/Float64 v
